src/main/fc/fc_msp.c

```diff
@@ -891,7 +929,12 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         sbufWriteU16(dst, 0);
 #endif
         sbufWriteU16(dst, sensors(SENSOR_ACC) | sensors(SENSOR_BARO) << 1 | sensors(SENSOR_MAG) << 2 | sensors(SENSOR_GPS) << 3 | sensors(SENSOR_SONAR) << 4);
-        sbufWriteU32(dst, packFlightModeFlags());
+        {
+            // pack only first 32 flags, use STATUS_EX to get them all
+            BITARRAY_TYPE(flightModeFlags, uint32_t, CHECKBOX_ITEM_COUNT);
```

Comment:
I believe configurator is currently using `MSP_STATUS` for modes tab, hence it will require more changes to it.